pennylane/transforms/optimization/cancel_inverses.py

```diff
@@ -30,12 +30,11 @@
 
 
 def _ops_equal(op1, op2):
-    """Checks if two operators are equal up to class, data, hyperparameters, and wires"""
+    """Checks if two operators are equal up to class, data, and hyperparameters"""
     return (
         op1.__class__ is op2.__class__
         and (op1.data == op2.data)
         and (op1.hyperparameters == op2.hyperparameters)
-        and (op1.wires == op2.wires)
     )
```

Comment:
Can we just use `qml.equal` now?

Reply:
`qml.equal` checks for wire equality as well, we want to compare all attributes _except_ for wires.

Reply:
Potentially rename this function to `_non_wires_equality`?

Reply:
Should we instead update our definition of "equal" to take into consideration operations that are symmetric?

Reply:
I changed the name of the function locally.

> Should we instead update our definition of "equal" to take into consideration operations that are symmetric?

I feel like this is a tangential discussion to object vs numerical equality. It certainly shouldn't be hard to integrate wire symmetry into `qml.equal` since we already have `Attribute`s containing a set of such ops. Maybe worth including more people in this discussion. Could be a potential "good first issue" if we decide it's worthwhile.